### ISSUE TITLE
Can't run `cargo doc` using stable rust?

### ISSUE THREAD

**Issue by vext01** [CensoredUsername/dynasm-rs#94]
Not sure if you were aware?

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> doc/insref/src/export.rs:1:1
  |
1 | #![feature(proc_macro_hygiene)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not document `insref`
```

Cheers.

**Comment by CensoredUsername** [comment]
huh, I don't recall that being an issue before. That is weird, and afaik shouldn't be necessary anymore.

**Comment by CensoredUsername** [comment]
Fixed in v3.0.0.
